test/test_automl/test_automl.py

```diff
@@ -64,9 +66,10 @@ def test_fit(dask_client):
         metric=accuracy,
         dask_client=dask_client,
     )
-    automl.fit(
-        X_train, Y_train, task=MULTICLASS_CLASSIFICATION
-    )
+
+    with ignore_warnings():
+        automl.fit(X_train, Y_train, task=MULTICLASS_CLASSIFICATION)
```

Comment:
Does the automl process really emit warnings? This would be very unfortunate.

Reply:
No it probably doesn't this was just a blanket sweep over some files wherever the method `fit` was called, this must have gotten swept up in it. I'll remove these occurences.